### PR TITLE
Fixes to LCA errors + BEIS managed assets elect EF

### PR DESCRIPTION
**Pull request by maryarbab**
**Is this a new addition or an update to an existing factor?**

- [*] New
- [*] Update

**What category or categories does it affect?**

_Multiple_

**What?**

_Fix some errors in the lca_activity and added one new emission factor (managed assets-electricity) from BEIS_

**Why?**

_Data debugging_

### CODE REVIEW

**Review by AnastasiaLobanova — APPROVED** [review]

**Review by AnastasiaLobanova — APPROVED** [review]

File: OpenEmissionFactorsDB.csv [hunk 1]
```diff
@@ -267,14 +267,14 @@ Consumer Goods and Services,Paper Products,paper_products-type_paper_paper_produ
...
-Consumer Goods and Services,Paper Products,paper_products-type_paper_sold,Paper sold (downstream),kg,,0.46,,,,,,,,downstream,GHG Protocol,2017,,,GLOBAL,,Climatiq,2021/10/12,Emission intensity of products sold: paper. Factors retrieved from the Quantis+GHG Protocol Scope 3 Calculator for the year 2016 and apply for years beyond 2016.,https://quantis-suite.com/Scope-3-Evaluator/
+Consumer Goods and Services,Paper Products,paper_products-type_paper_sold,Paper sold (downstream),kg,,0.46,,,,,,,,unknown,GHG Protocol,2017,,,GLOBAL,,Climatiq,2021/10/12,Emission intensity of products sold: paper. Factors retrieved from the Quantis+GHG Protocol Scope 3 Calculator for the year 2016 and apply for years beyond 2016.,https://quantis-suite.com/Scope-3-Evaluator/
```
Comment (patstirling) [inline OpenEmissionFactorsDB.csv:270]: this one got missed on the data infill I think!

File: OpenEmissionFactorsDB.csv [hunk 2]
```diff
@@ -2580,15 +2581,15 @@ Materials,Mined Materials,mined_materials-type_subbituminous_coal,Sub-bituminous
...
-Materials,Organic Products,organic_products-type_organics_sold,Organics sold (downstream),kg,,0.31,,,,,,,,downstream,GHG Protocol,2017,,,GLOBAL,,Climatiq,2021/10/12,Emission intensity of products sold: organics. Factors retrieved from the Quantis+GHG Protocol Scope 3 Calculator for the year 2016 and apply for years beyond 2016.,https://quantis-suite.com/Scope-3-Evaluator/
+Materials,Organic Products,organic_products-type_organics_sold,Organics sold (downstream),kg,,0.31,,,,,,,,unknown,GHG Protocol,2017,,,GLOBAL,,Climatiq,2021/10/12,Emission intensity of products sold: organics. Factors retrieved from the Quantis+GHG Protocol Scope 3 Calculator for the year 2016 and apply for years beyond 2016.,https://quantis-suite.com/Scope-3-Evaluator/
```
Comment (patstirling) [inline OpenEmissionFactorsDB.csv:2592]: This one too

File: OpenEmissionFactorsDB.csv [hunk 3]
```diff
@@ -2597,7 +2598,7 @@ Materials,Other Materials,other_materials-type_additives_blending_components,Add
...
-Materials,Other Materials,other_materials-type_mixed_materials_sold,Mixed materials sold (downstream),kg,,0.53,,,,,,,,downstream,GHG Protocol,2017,,,GLOBAL,,Climatiq,2021/10/12,Emission intensity of products sold: mixed materials . Factors retrieved from the Quantis+GHG Protocol Scope 3 Calculator for the year 2016 and apply for years beyond 2016.,https://quantis-suite.com/Scope-3-Evaluator/
+Materials,Other Materials,other_materials-type_mixed_materials_sold,Mixed materials sold (downstream),kg,,0.53,,,,,,,,unknown,GHG Protocol,2017,,,GLOBAL,,Climatiq,2021/10/12,Emission intensity of products sold: mixed materials . Factors retrieved from the Quantis+GHG Protocol Scope 3 Calculator for the year 2016 and apply for years beyond 2016.,https://quantis-suite.com/Scope-3-Evaluator/
```
Comment (patstirling) [inline OpenEmissionFactorsDB.csv:2601]: This one

File: OpenEmissionFactorsDB.csv [hunk 4]
```diff
@@ -2651,7 +2652,7 @@ Materials,Plastics and Rubber Products,plastics_rubber-type_pet_including_formin
...
-Materials,Plastics and Rubber Products,plastics_rubber-type_plastics_sold,Plastics sold (downstream),kg,,0.04,,,,,,,,downstream,GHG Protocol,2017,,,GLOBAL,,Climatiq,2021/10/12,Emission intensity of products sold: plastics. Factors retrieved from the Quantis+GHG Protocol Scope 3 Calculator for the year 2016 and apply for years beyond 2016.,https://quantis-suite.com/Scope-3-Evaluator/
+Materials,Plastics and Rubber Products,plastics_rubber-type_plastics_sold,Plastics sold (downstream),kg,,0.04,,,,,,,,unknown,GHG Protocol,2017,,,GLOBAL,,Climatiq,2021/10/12,Emission intensity of products sold: plastics. Factors retrieved from the Quantis+GHG Protocol Scope 3 Calculator for the year 2016 and apply for years beyond 2016.,https://quantis-suite.com/Scope-3-Evaluator/
```
Comment (patstirling) [inline OpenEmissionFactorsDB.csv:2655]: This one

File: OpenEmissionFactorsDB.csv [hunk 5]
```diff
@@ -2680,11 +2681,11 @@ Materials,Primary Metals,metals_primary-type_alumina_refining_primary_aluminum_p
...
-Materials,Primary Metals,metals_primary-type_metal_sold,Metal sold (downstream),kg,,0.04,,,,,,,,downstream,GHG Protocol,2017,,,GLOBAL,,Climatiq,2021/10/12,Emission intensity of products sold: metal. Factors retrieved from the Quantis+GHG Protocol Scope 3 Calculator for the year 2016 and apply for years beyond 2016.,https://quantis-suite.com/Scope-3-Evaluator/
+Materials,Primary Metals,metals_primary-type_metal_sold,Metal sold (downstream),kg,,0.04,,,,,,,,unknown,GHG Protocol,2017,,,GLOBAL,,Climatiq,2021/10/12,Emission intensity of products sold: metal. Factors retrieved from the Quantis+GHG Protocol Scope 3 Calculator for the year 2016 and apply for years beyond 2016.,https://quantis-suite.com/Scope-3-Evaluator/
```
Comment (patstirling) [inline OpenEmissionFactorsDB.csv:2688]: This one

File: OpenEmissionFactorsDB.csv [hunk 6]
```diff
@@ -3190,8 +3191,8 @@ Transport,Air Travel,passenger_flight-route_type_na-aircraft_type_na-distance_na
...
+Transport,Air Travel,passenger_flight-route_type_na-aircraft_type_na-distance_na-class_na-rf_na,Airplane - RF effect unknown,km,,0.177,,,,,,,,unknown,GHG Protocol,2017,,,GLOBAL,,Climatiq,2021/10/12,Emission intensity of supply chain (purchase of service/standard good/capital good) in in kilometers travelled by: airplane. Factors retrieved from the Quantis+GHG Protocol Scope 3 Calculator for the year 2016 and apply for years beyond 2016. The same emission factor applies to different types of investments in joint ventures/subsidiaries/associate companies that were not captured in Scope 1 emissions. NOTE: It is unknown if the radiative forcing (RF) effect that reflects the impact of contrails and high-altitude release of GHGs is included.,https://quantis-suite.com/Scope-3-Evaluator/
```
Comment (patstirling) [inline OpenEmissionFactorsDB.csv:3195]: This one

File: OpenEmissionFactorsDB.csv [hunk 7]
```diff
@@ -3347,8 +3348,8 @@ Transport,Rail Travel,passenger_train-route_type_national_rail-fuel_source_na,Na
...
+Transport,Rail Travel,passenger_train-route_subway-fuel_source_na,Subway,km,,0.066,,,,,,,,unknown,GHG Protocol,2017,,,GLOBAL,,Climatiq,2021/10/12,Emission intensity of supply chain (purchase of service/standard good/capital good) in kilometers travelled by: subway. Factors retrieved from the Quantis+GHG Protocol Scope 3 Calculator for the year 2016 and apply for years beyond 2016. The same emission factor applies to different types of investments in joint ventures/subsidiaries/associate companies that were not captured in Scope 1 emissions.,https://quantis-suite.com/Scope-3-Evaluator/
```
Comment (patstirling) [inline OpenEmissionFactorsDB.csv:3352]: This one

File: OpenEmissionFactorsDB.csv [hunk 8]
```diff
@@ -3508,24 +3509,24 @@ Transport,Road Freight,freight_vehicle-vehicle_type_light_delivery-fuel_source_p
...
+Transport,Road Travel,passenger_vehicle-vehicle_type_automobile-fuel_source_na-distance_na-engine_size_na,Automobile,km,,0.463,,,,,,,,unknown,GHG Protocol,2017,,,GLOBAL,,Climatiq,2021/10/12,Emission intensity of supply chain (purchase of service/standard good/capital good) in kilometers travelled by: automobile. Factors retrieved from the Quantis+GHG Protocol Scope 3 Calculator for the year 2016 and apply for years beyond 2016. The same emission factor applies to different types of investments in joint ventures/subsidiaries/associate companies that were not captured in Scope 1 emissions.,https://quantis-suite.com/Scope-3-Evaluator/
```
Comment (patstirling) [inline OpenEmissionFactorsDB.csv:3513]: This one

File: OpenEmissionFactorsDB.csv [hunk 9]
```diff
@@ -3508,24 +3509,24 @@ Transport,Road Freight,freight_vehicle-vehicle_type_light_delivery-fuel_source_p
...
+Transport,Road Travel,passenger_vehicle-vehicle_type_bicycle-fuel_source_na-distance_na-engine_size_na,Bicycle,km,,0,,,,,,,,unknown,GHG Protocol,2017,,,GLOBAL,,Climatiq,2021/10/12,Emission intensity of supply chain (purchase of service/standard good/capital good) in kilometers travelled by: bicycle. Factors retrieved from the Quantis+GHG Protocol Scope 3 Calculator for the year 2016 and apply for years beyond 2016. The same emission factor applies to different types of investments in joint ventures/subsidiaries/associate companies that were not captured in Scope 1 emissions.,https://quantis-suite.com/Scope-3-Evaluator/
```
Comment (patstirling) [inline OpenEmissionFactorsDB.csv:3522]: This one

File: OpenEmissionFactorsDB.csv [hunk 10]
```diff
@@ -3508,24 +3509,24 @@ Transport,Road Freight,freight_vehicle-vehicle_type_light_delivery-fuel_source_p
...
+Transport,Road Travel,passenger_vehicle-vehicle_type_bus-fuel_source_na-distance_na-engine_size_na,Bus,km,,0.1234,,,,,,,,unknown,GHG Protocol,2017,,,GLOBAL,,Climatiq,2021/10/12,Emission intensity of supply chain (purchase of service/standard good/capital good) in kilometers travelled by: bus. Factors retrieved from the Quantis+GHG Protocol Scope 3 Calculator for the year 2016 and apply for years beyond 2016. The same emission factor applies to different types of investments in joint ventures/subsidiaries/associate companies that were not captured in Scope 1 emissions.,https://quantis-suite.com/Scope-3-Evaluator/
```
Comment (patstirling) [inline OpenEmissionFactorsDB.csv:3529]: this one

File: OpenEmissionFactorsDB.csv [hunk 11]
```diff
@@ -3584,10 +3585,10 @@ Transport,Road Travel,passenger_vehicle-vehicle_type_car-fuel_source_petrol_phev
...
+Transport,Road Travel,passenger_vehicle-vehicle_type_taxi-fuel_source_na-distance_na-engine_size_na,Taxi,km,,0.2751,,,,,,,,unknown,GHG Protocol,2017,,,GLOBAL,,Climatiq,2021/10/12,Emission intensity of supply chain (purchase of service/standard good/capital good) in kilometers travelled by: taxi. Factors retrieved from the Quantis+GHG Protocol Scope 3 Calculator for the year 2016 and apply for years beyond 2016. The same emission factor applies to different types of investments in joint ventures/subsidiaries/associate companies that were not captured in Scope 1 emissions.,https://quantis-suite.com/Scope-3-Evaluator/
```
Comment (patstirling) [inline OpenEmissionFactorsDB.csv:3589]: this one

File: OpenEmissionFactorsDB.csv [hunk 12]
```diff
@@ -3584,10 +3585,10 @@ Transport,Road Travel,passenger_vehicle-vehicle_type_car-fuel_source_petrol_phev
...
+Transport,Road Travel,passenger_vehicle-vehicle_type_van-fuel_source_na-distance_na-engine_size_na,Van,km,,0.3218,,,,,,,,unknown,GHG Protocol,2017,,,GLOBAL,,Climatiq,2021/10/12,Emission intensity of supply chain (purchase of service/standard good/capital good) in kilometers travelled by: van. Factors retrieved from the Quantis+GHG Protocol Scope 3 Calculator for the year 2016 and apply for years beyond 2016. The same emission factor applies to different types of investments in joint ventures/subsidiaries/associate companies that were not captured in Scope 1 emissions.,https://quantis-suite.com/Scope-3-Evaluator/
```
Comment (patstirling) [inline OpenEmissionFactorsDB.csv:3591]: this one

File: OpenEmissionFactorsDB.csv [hunk 13]
```diff
@@ -3684,10 +3685,10 @@ Transport,Sea Freight,sea_freight-vessel_type_container_gt_3000teu_lt_4999teu-ro
...
-Transport,Sea Freight,sea_freight-vessel_type_cargo-route_type_inland_western_europe-vessel_length_gt_163m_lt_185m-tonnage_na-fuel_source_na,Coupled convoy 163 - 185m,tonne-km,,0.017,,,,,,,,upstream-fuel_combustion,STC-Nestra B.V.,2018,,,EU,,Climatiq,2021/09/09,Emission intensity for a coupled convoy (between 163 and 185 meters). The indicated value corresponds to the Amsterdam-Rotterdam-Antwerp and Rhine routes interchangeably. Published by STC-NESTRA B.V in 2018.,https://www.smartfreightcentre.org/pdf/GLEC-report-on-GHG-Emission-Factors-for-Inland-Waterways-Transport-SFC2018.pdf
+Transport,Sea Freight,sea_freight-vessel_type_cargo-route_type_inland_western_europe-vessel_length_110m-tonnage_na-fuel_source_na,Container vessels 110m,tonne-km,,0.0255,,,,,,,,well_to_propeller,STC-Nestra B.V.,2018,,,EU,,Climatiq,2021/09/08,Emission intensity for a container vessels. Amsterdam-Rotterdam-Antwerp route. Published by STC-NESTRA B.V in 2018.,https://www.smartfreightcentre.org/pdf/GLEC-report-on-GHG-Emission-Factors-for-Inland-Waterways-Transport-SFC2018.pdf
```
Comment (patstirling) [inline OpenEmissionFactorsDB.csv:3688]: these 4

File: OpenEmissionFactorsDB.csv [hunk 14]
```diff
@@ -3739,11 +3740,11 @@ Transport,Sea Freight,sea_freight-vessel_type_lpg_tanker_gt_50000m3-route_type_n
...
-Transport,Sea Freight,sea_freight-vessel_type_cargo-route_type_inland_western_europe-vessel_length_gt_87m_lt_109m-tonnage_na-fuel_source_na,Motor vessel 87 - 109m,tonne-km,,0.0184,,,,,,,,upstream-fuel_combustion,STC-Nestra B.V.,2018,,,EU,,Climatiq,2021/09/11,Emission intensity for motor vessel (between 87 and 109 meters). Western Europe route. Published by STC-NESTRA B.V in 2018.,https://www.smartfreightcentre.org/pdf/GLEC-report-on-GHG-Emission-Factors-for-Inland-Waterways-Transport-SFC2018.pdf
+Transport,Sea Freight,sea_freight-vessel_type_container_coupled_convoy-route_type_inland_western_europe_route-vessel_length_lte_80m-tonnage_na-fuel_source_na,Motor vessel ≤ 80m,tonne-km,,0.0295,,,,,,,,well_to_propeller,STC-Nestra B.V.,2018,,,EU,,Climatiq,2021/09/14,Emission intensity for motor vessel (Less than or equal to 80 meters). Western Europe route. Published by STC-NESTRA B.V in 2018.,https://www.smartfreightcentre.org/pdf/GLEC-report-on-GHG-Emission-Factors-for-Inland-Waterways-Transport-SFC2018.pdf
```
Comment (patstirling) [inline OpenEmissionFactorsDB.csv:3743]: these 5

**Review by patstirling — CHANGES_REQUESTED** [review]

**Review by patstirling — APPROVED** [review]

**Pull request merged**